### DISCUSSION TITLE
"Dfsu - Export to netcdf.ipynb" produces a .nc file that does not similar to .dsfu file?

### DISCUSSION THREAD

**Discussion by nguyenduy1133** [DHI/mikeio#466] [Q&A] [answered]
Dear all,

I ran the code Dfsu - Export to netcdf.ipynb and produced *.nc output file. However, when I open this file in QGIS and compare it with the original *.dfsu file, I see that the two files are not the same.
I want to create .nc file with X, Y axis as coordinates, display value is a variable like salinity.
Please help me how to do it?

Yours sincerely,
ND Liem

**Comment by ecomodeller** [comment, marked as answer]
Have you looked at this notebook <https://dhi.github.io/mikeio-examples/notebooks/Dfsu_-_2D_interpolation.html>?
